test/dlc_tests/conftest.py

```diff
@@ -276,9 +311,10 @@ def pytest_generate_tests(metafunc):
                             images_to_parametrize.append(image)
                         elif "gpu_only" in metafunc.fixturenames and "gpu" in image:
                             images_to_parametrize.append(image)
-                        elif "cpu_only" not in metafunc.fixturenames and "gpu_only" not in metafunc.fixturenames:
+                        elif "eia_only" in metafunc.fixturenames and ("cpu" in image or "eia" in image):
```

Comment:
Why check if "cpu" in image?

This will parametrize the image_uri in EI tests to include CPU images as well, right?

Reply:
It should be safe to remove CPU in this line. Although in this PR, it is running the EIA test only and skipping non eia test as expected and in #355, which is the same code but running cpu and gpu image, all the eia test are been skipped.